modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc

```diff
@@ -206,20 +206,23 @@ class NDB_Menu_Filter_Form_Conflict_Resolver extends NDB_Menu_Filter_Form
 
         // set the class variables
         $this->columns = array(
-                          'conflicts_unresolved.TableName',
-                          'session.CandID',
-                          'candidate.PSCID',
-                          'session.Visit_label',
+                          'conflicts_unresolved.TableName as Instrument',
+                          'session.CandID as CandID',
+                          'candidate.PSCID as PSCID',
+                          'session.Visit_label as Visit_Lable',
```

Comment:
As a matter of fact, none of the aliases assigned in this piece of code are used anywhere, so there's no reason to add them